packages/autocomplete-html/update/update.js

```diff
@@ -0,0 +1,289 @@
+                    source code. Where values are `html`, `dom`, `cssom`.
+                    We likely will not find any use here.
+  To view the structure of this file visit:
+  https://github.com/ChromeDevTools/devtools-frontend/blob/main/front_end/models/javascript_metadata/DOMPinnedProperties.ts
+  ==============================================================================
+  ==============================================================================
+  Finally with the topics covered above we are able to collect all the data needed.
+  The only value that is not being retreived or covered here is:
+  - `type`: The type defined within our `completions` attributes. Now the global
+  attribute could be retreived by looking for the value from the DevTools data
+  or could be known that when we find MDNs documentation for it, the global
+  attribute docs reside in a different folder than the rest of the docs. But
+  for all other types we may need to keep a manual list of where it applies.
+  Most types only apply to a single attribute so would be simple, but
+  `boolean` and `flag` types may have to be manually maintained.
+  - `attribOption`: The valid values to pass an attribute currently
+  does not have any consistent method to retreive valid values.
+  Ideally this can be found within the `@webref` data, or alternatively may be
+  collected from MDNs `web/html/attributes` documents. But may prove to be difficult
+  to reliably parse.
+
+  A note is that further research shows that MDNs attributes documentation cannot
+  be created the same way as one would expect. That is by the folder names
+  within the `web/html/attributes` folder. Instead we may have to parse
+  a table of values from `attributes/index.md` as that contains many items and links
+  to unwritten pages.
+  ==============================================================================
+  ==============================================================================
+  The last note of our implementation.
+  The provided attributes used within our `completions.json` seem interesting.
+  The majority of them are only attributes that are not suggested via tags we have.
+  Additionally all of the information, like mentioned earlier, would be rather
+  difficult to automatically collect. Since we have no reliable way to obtain `type`,
+  `global` values. With only the `description` being acheivable. But considering
+  that this list is non-exhustive, and is explicitely items not included above
+  it leads me to beleive that this list has been manually curated since inception.
+  And rather than attempt to manually curate all of it's unique parts with only
+  descriptions and global status being manual, while increasing the size ten-fold
+  to include all possible attributes, it may be best to leave as is, and reinsert
+  it into our completions during update time.
+ */
+
+const chromiumElementsShim = require("./chromium-elements-shim.js");
```

Comment:
We can probably get rid of this `.js` that delegates to `.mjs` if we use `import` in place of `require`.

`import` will happily accept a ESM inside a CommonJS, and the result will be a promise, so we just need to `await` for it to resolve... like so:

```suggestion
const chromiumElementsShim = import("./chromium-elements-shim.mjs");
```

Reply:
I'll try this. Although I think I did something similar and it was failing. But that was likely before I had implemented the translation from TypeScript

Reply:
I'll try this. Although I think I did something similar and it was failing. But that was likely before I had implemented the translation from TypeScript

Reply:
As for this one, doing some more research after your suggestion was failing "CommonJS modules cannot import ES Modules"

So it seems we might have to keep this hacky way to keep things going. Plus because the package we use to convert TypeScript on the fly is also an ESM module. So unfortunately we might not be able to fix this right away.